src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -1569,7 +1550,8 @@ class DropTableStatementSegment(BaseSegment):
         "TABLE",
         Ref("IfExistsGrammar", optional=True),
         Ref("TableReferenceSegment"),
-        OneOf("RESTRICT", Ref.keyword("CASCADE", optional=True), optional=True),
+        # OneOf("RESTRICT", Ref.keyword("CASCADE", optional=True), optional=True),
```

Comment:
remove the comment

Reply:
also type of this class should be updated to `type = "drop_table_statement"`

Reply:
√